plugin.rb

```diff
@@ -216,16 +217,16 @@ def self.onebox_template
       limited_pretty_text_markdown_rules: ChatMessage::MARKDOWN_IT_RULES,
     }
 
-    Guardian.class_eval { include DiscourseChat::GuardianExtensions }
-    UserNotifications.class_eval { prepend DiscourseChat::UserNotificationsExtension }
-    UserOption.class_eval { prepend DiscourseChat::UserOptionExtension }
-    Category.class_eval { has_one :chat_channel, as: :chatable }
+    Guardian.prepend DiscourseChat::GuardianExtensions
+    UserNotifications.prepend DiscourseChat::UserNotificationsExtension
+    UserOption.prepend DiscourseChat::UserOptionExtension
+    Category.prepend DiscourseChat::CategoryExtension
     User.class_eval do
```

Comment:
I guess we could probably normalize the use of `prepend` with a module as I think it would be better from a code organization point of view.

Reply:
I did it where we only had a call `prepend` or where I had to change something. I think we should probably do it for `User` for example (but I didn’t as it’s not necessarily related to this PR) :slightly_smiling_face:

Reply:
Ok cool, free to do it in a follow up PR thanks